src/sinks/humio/metrics.rs

```diff
@@ -252,11 +253,11 @@ mod tests {
 
         let output = rx.take(len).collect::<Vec<_>>().await;
         assert_eq!(
-            r#"{"event":{"counter":{"value":42.0},"kind":"incremental","name":"metric1","tags":{"os.host":"somehost"}},"fields":{},"time":1597784401.0}"#,
+            r#"{"event":{"counter":{"value":42.0},"kind":"incremental","name":"metric1","tags":{"os.host":"somehost"},"timestamp":"2020-08-18T21:00:01Z"},"fields":{}}"#,
```

Comment:
I think this isn't actually quite right. `time` should continue to be a top level field that is a unix timestamp (with fractional seconds). See the example Splunk has here for sending metrics: https://docs.splunk.com/Documentation/Splunk/8.0.1/Metrics/GetMetricsInOther#:~:text=Reference%20Manual.-,Example%20of%20sending%20metrics%20using%20HEC,-The%20following%20example

Reply:
Good catch... I shouldn't have changed the test to fit the bug :)

Reply:
Noting we discussed this and discovered that the `humio_metrics` sink actually sends metrics in a different format (which I can't find the docs for), but we do want to preserve the timestamp behavior here.